### DISCUSSION TITLE
Branding folder - global settings - no luck.

### DISCUSSION THREAD

**Discussion by balkib** [filebrowser/filebrowser#1984] [General]
I referenced this: https://filebrowser.org/configuration/custom-branding
And this:
https://github.com/filebrowser/filebrowser/tree/master/frontend/public/img

1) Copied the icons folder from Github. Replaced all files with altered branding, but retained same file names.

2) Set Global Setting -> Branding directory path as: /tmp/Branding/icons/

Doesn't apply new branding. Not sure what I'm missing. Thoughts would be helpful.

**Comment by ramiresviana** [comment]
When the `Branding directory path` option is set to a directory containing `img/icons` folders, File Browser will use that directory for the icons.
